client/sites/hooks/use-initialize-dataviews-page.ts

```diff
@@ -1,6 +1,6 @@
 import { usePrevious } from '@wordpress/compose';
 import { useEffect, useRef } from 'react';
-import type { View } from '@wordpress/dataviews';
+import type { View } from '@automattic/dataviews';
```

Comment:
I guess this change is fine. I'd like to know whether we want to change this, considering we may need to go back upstream in the future. cc @oandregal

Reply:
I changed this because lint/CI issues were reported. Dotcom Site is using `@automattic/dataviews`. Using `@wordpress/dataviews` will cause incompatible types.

https://github.com/Automattic/wp-calypso/blob/10f282589208f4270d6a93fbd9c9a59fc443d2b2/client/sites/components/sites-dataviews/index.tsx#L2